cmd/claimserver/config.example.yaml

```diff
@@ -0,0 +1,19 @@
+server:
+	serviceapi: 0.0.0.0:6000
```

Comment:
Inconsistent indentation.  Use 2 spaces like in the rest of this file.

Reply:
done